src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/PlatformManifestReader.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    static class PlatformManifestReader
+    {
+        static readonly char[] s_manifestLineSeparator = new[] { '|' };
+        public static IEnumerable<ConflictItem> LoadConflictItems(string manifestPath, ILog log)
+        {
+            if (manifestPath == null)
+            {
+                throw new ArgumentNullException(nameof(manifestPath));
+            }
+
+            if (!File.Exists(manifestPath))
+            {
+                string errorMessage = string.Format(CultureInfo.InvariantCulture, Strings.CouldNotLoadPlatformManifest,
```

Comment:
(suggestion) I find an extension method on string named `FormatInvariantCulture` works well to cut down on duplicating this everywhere.